### PR TITLE
(DOCSP-11872): Add limitation for updating sharded collection

### PR DESCRIPTION
**Pull request by jeff-allen-mongo**
Consolidated the Read-only limitation and this new limitation to a new `Limitations` section at the top of the page.

See staging: [Modify Documents](https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-11872/documents/modify.html)

### CODE REVIEW

**Comment by jeff-allen-mongo** [comment]
@mmarcon, these are the updates for the sharded collection limitation discussed over Slack. Let me know if you'd like to see anything different. Thanks!

File: source/documents/modify.txt [hunk 1]
```diff
@@ -6,16 +6,29 @@ Modify Documents
...
-
-   Modifying documents is not permitted in
-   :guilabel:`MongoDB Compass Readonly Edition`.
-
 You can edit existing documents in your collection. When you edit
 a document, |compass-short| performs a
 :manual:`findAndModify </reference/method/db.collection.findAndModify/>`
 operation to update the document.
 
+Limitation
```
Comment (mmarcon) [inline source/documents/modify.txt:14]:
```suggestion
Limitations
```

**Review by mmarcon — APPROVED** [review]
Left a minor suggestion, which might be wrong as I am not a native speaker.

Other than that, looks great!

File: source/documents/modify.txt (hunk 1 above)
Comment (jeff-allen-mongo) [reply source/documents/modify.txt:14]: Yes should definitely be `Limitations`, my mistake. Thanks!

**Pull request merged**